docs/source/CONTRIBUTING.md

```diff
@@ -109,9 +109,9 @@ to get used to the basic usage.
 
 Now, setting up a development environment is super simple! Additional info if you run into trouble: [Poetry Environments]
 
-Step 1: [Install Poetry]  
-Step 2: Run ``poetry install``  
-Step 3: Optionally Run ``poetry shell``  
+Step 1: [Install Poetry]
+Step 2: Run ``poetry install``
+Step 3: Optionally Run ``poetry shell``
```

Comment:
Is there a way we can ignore Markdown files in the whitespace fixer? In Markdown, these are actually significant. Two spaces at the end of a line are used to indicate a line break, so removing these spaces breaks the formatting here.

Reply:
If you want to see it locally, you can `poetry install -E docs` then `poetry run make -C docs html` and view `docs/build/html/CONTRIBUTING.html`